extensions/blocks/simple-payments/edit.js

```diff
@@ -524,7 +524,8 @@ class SimplePaymentsEdit extends Component {
 						onChange={ this.handleEmailChange }
 						placeholder={ __( 'Email', 'jetpack' ) }
 						required
-						type="email"
+						// TODO: switch this back to type="email" once Gutenberg paste handler ignores inputs of type email
```

Comment:
Age-old PR but came up in my search when looking for something else...

You could use `inputmode="email"` meanwhile to get the right keyboard on mobile. Otherwise, the type doesn't matter that much.

https://css-tricks.com/everything-you-ever-wanted-to-know-about-inputmode/